### PR TITLE
Add shaunferns26 to github graduation

### PR DESCRIPTION
**Pull request by shaunferns26**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

File: _data/shaunferns26/shaunferns26.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Shaun
```
Comment (Mohammed-Shoaib) [inline _data/shaunferns26/shaunferns26.md:2]: Use your full name please, "Shaun Fernandes" or "Shoon-bot" whichever suits you better.

File: _data/shaunferns26/shaunferns26.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+institution: BITS Pilani
```
Comment (Mohammed-Shoaib) [inline _data/shaunferns26/shaunferns26.md:3]: Use "Birla Institute of Technology and Science, Pilani -- Dubai", fits the 58-character limit mark _perfectly_. If you want you can remove the _Dubai_ if you'd like :D

File: _data/shaunferns26/shaunferns26.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+profile_pic: octocat.png
```
Comment (Mohammed-Shoaib) [inline _data/shaunferns26/shaunferns26.md:4]: Although that definitely looks like the suit you have and the shoes are pretty similar too, please use your own picture!

**Comment by Mohammed-Shoaib** [comment]
Thanks for sending the PR Shoooooooooon! :heart:
Yeah man, you missed out on all the amazing swag. :disappointed:
I like how you used the automated reply from reddit, which really makes you look like a bot. :robot:

**Comment by Mohammed-Shoaib** [comment]
@shaunferns26 Yaaaaaaaaaaaaay! :tada: :confetti_ball:
Welcome to the yearbook bot. :robot:

**Comment by shaunferns26** [comment]
Hey @juanpflores. Thanks for merging the PR. No, I didn't submit a form, the deadline had already passed by the time I read about GitHub graduation.

**Pull request merged**